torch/_functorch/partitioners.py

```diff
@@ -1879,12 +1910,12 @@ def classify_nodes(joint_module):
 
         # Log theoretical per activation storage sizes
         log.info("Theoretical Per Activation Storage Sizes: %s", sorted_sizes)
-        fw_module_nodes = {
-            node.name for node in fw_module.graph.nodes if node.op == "call_function"
-        }
-        bw_module_nodes = {
-            node.name for node in bw_module.graph.nodes if node.op == "call_function"
-        }
+        fw_module_nodes = OrderedSet(
+            [node.name for node in fw_module.graph.nodes if node.op == "call_function"]
```

Comment:
These list comprehensions can be generators if you want an extra copy (input only needs to be an Iterable)